### DISCUSSION TITLE
Tutoriel en Français (ou sous-titré)

### DISCUSSION THREAD

**Discussion by MartinWINDELS** [birdnet-team/BirdNET-Analyzer#538] [General]
Bonsoir, 
J'utilise BirdNET Analyzer depuis peu. Je voudrais gagner du temps à l'analyse d'un enregistrement de nuit. 
Je viens de comparer un enregistrement court où j'ai, à l'oreille, identifié les "intervenants" chants ou cris. J'ai chipoté dans les différents paramètres mais je comprends de moins en moins. 
J'ai par ex. une corneille noire qui crie et elle n'est apparue qu'une seule reprise mais je n'avais pas noté les paramètres. Maintenant, je ne retrouve pas comment je peux retrouver ces réglages. 
A vous lire.
Sinon, meilleurs voeux à toutes et tous.

**Comment by Mattk70** [comment]
Pour l'analyse des enregistrements nocturnes, vous pourriez essayer [Chirpity](https://chirpity.mattkirkland.co.uk). Il dispose d'une aide détaillée (en français et dans 10 autres langues), vous permet d'utiliser BirdNET pour l'analyse, mais possède également un modèle dédié spécialement optimisé pour les appels Nocmig.

**Comment by MartinWINDELS** [comment]
Bonjour,
Merci pour votre retour. Les critiques sont très négatives et pourtant je
trouve qu'il me facilite le travail et fait appel à BirdNET. Pour
l'utilisation de BirdNET ANALYZER, j'ai trouvé la bonne programmation mais
je ne peux pas (aisément) aller vérifier les cris et ça me déplait
fortement.
A+
Hello,
Thanks for your feedback. Regarding CHIRPITY, the reviews are very negative
and yet I find that it makes my work easier and uses BirdNET. For the use
of BirdNET ANALYZER, I found the right programming but I can't (easily) go
and check the cries and I really don't like that.
Best regards.

Le jeu. 2 janv. 2025 à 23:50, Mattk70 ***@***.***> a écrit :

> Pour l'analyse des enregistrements nocturnes, vous pourriez essayer
> Chirpity <https://chirpity.mattkirkland.co.uk>. Il dispose d'une aide
> détaillée (en français et dans 10 autres langues), vous permet d'utiliser
> BirdNET pour l'analyse, mais possède également un modèle dédié spécialement
> optimisé pour les appels Nocmig.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kahst/BirdNET-Analyzer/discussions/538#discussioncomment-11721192>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BNRRO3PMZXCOKWHYZSJ5CAD2IW7DDAVCNFSM6AAAAABUMLVECWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNZSGEYTSMQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 


*Martin WINDELS*
*7780 COMINES, chaussée de Wervicq, 198*
***@***.**** ***@***.***>
*+32(0)474869508*

*MEMBRE CNB, NATAGORA, AVES, AVES MONS-TOURNAI, LRBPO, NATUURPUNT,
GON...MEMBRE du RESEAU NATURE : **1068P*
« Le silence des bêtes est la double expression de leur dignité et de notre
déshonneur. Nous autres, humains, faisons tant de vacarme. » Sylvain Tesson
« Je ne veux pas protéger la nature mais créer un monde où la nature
n'aurait pas besoin d'être protégée. »
« Mon pays est le monde et ma religion est de faire le bien. »  Thomas
Paine
J'y ajoute que mon peuple est l'Humanité.
« Le monde ne mourra pas par manque de merveilles mais uniquement par
manque d'émerveillement. »
« Ne doutez jamais qu'un petit groupe de gens réfléchis et engagés puisse
changer le monde. En fait, c'est toujours comme cela que ça s'est
passé. »  Margaret
Mead (1901 - 1978) - Anthropologue.

**Reply by Mattk70** [reply]
Oh, it's disappointing to hear Chirpity has negative reviews. As it's developer, I'd like to know what people think of it. The only reviews I've seen are positive, like your own experience. Perhaps you could let me know more about these reviews - you could message me on the Chirpity WhatsApp channel if you like.

**Comment by MartinWINDELS** [comment]
Hello,
When I talked about the use of CHIRPITY, at various Forums on night
migration, the criticisms come mainly from the fact that the AI does not
detect all the cries and is often wrong. So
Does not detect all?? (I will ask what is not detected)
Is wrong: here, I believe it is because the AI puts a name on everything.
My dogs, for example, have already had many different bird names.
Personally, while not knowing if all the cries have been detected and
identified on the recording, I check all detected cries. I do it in two
ways: if visually, it can match, I accept it, the other way is by hearing.
If I, by hearing, do not hear the cry and I do not have visual
confirmation, I do not accept it.
I take this opportunity to ask you questions. You have of course the
opportunity to answer or not and/or to recommend a tutorial or something
else.
What is the button with the two arrows at the bottom right of the sonogram
(third from the left) for please?
Sometimes the program takes a range but cuts out a noise or cry badly and
therefore falsifies the identification itself. Am I clear?
When following the situation above, I adjust the range myself and create a
new example, is it useful to you or not) does it refine the AI on my PC
only or possibly is it useless?
I prefer to work by email and I have little knowledge of computers (I am 66
years old).
Hoping not to have been too long.
I use an online translator.
Best regards.

Le ven. 10 janv. 2025 à 20:58, Mattk70 ***@***.***> a écrit :

> Oh, it's disappointing to hear Chirpity has negative reviews. As it's
> developer, I'd like to know what people think of it. The only reviews I've
> seen are positive, like your own experience. Perhaps you could let me know
> more about these reviews - you could message me on the Chirpity WhatsApp
> channel if you like.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kahst/BirdNET-Analyzer/discussions/538#discussioncomment-11802038>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BNRRO3K35EGFOLSCFCAN7BT2KAQ5TAVCNFSM6AAAAABUMLVECWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCOBQGIYDGOA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 


*Martin WINDELS*
*7780 COMINES, chaussée de Wervicq, 198*
***@***.**** ***@***.***>
*+32(0)474869508*

*MEMBRE CNB, NATAGORA, AVES, AVES MONS-TOURNAI, LRBPO, NATUURPUNT,
GON...MEMBRE du RESEAU NATURE : **1068P*
« Le silence des bêtes est la double expression de leur dignité et de notre
déshonneur. Nous autres, humains, faisons tant de vacarme. » Sylvain Tesson
« Je ne veux pas protéger la nature mais créer un monde où la nature
n'aurait pas besoin d'être protégée. »
« Mon pays est le monde et ma religion est de faire le bien. »  Thomas
Paine
J'y ajoute que mon peuple est l'Humanité.
« Le monde ne mourra pas par manque de merveilles mais uniquement par
manque d'émerveillement. »
« Ne doutez jamais qu'un petit groupe de gens réfléchis et engagés puisse
changer le monde. En fait, c'est toujours comme cela que ça s'est
passé. »  Margaret
Mead (1901 - 1978) - Anthropologue.

**Reply by Mattk70** [reply]
@MartinWINDELS, Merci pour votre explication.

Lorsque vous analysez un long fichier audio, les modèles d'IA sont souvent désorientés, surtout dans les environnements urbains où il y a beaucoup de bruits de fond. De plus, les appels faibles ou inhabituels peuvent ne pas être détectés correctement. La fonctionnalité clé de Chirpity est qu'elle facilite grandement l'examen, le filtrage et la vérification des détections. Vous pouvez également comparer ce que l'IA pense être les cris avec une sélection des cris similaires provenant du site web Xeno-Canto.

Je pense que vous parlez de ce bouton pour sélectionner une plage :
![image](https://github.com/user-attachments/assets/f76889aa-4796-4104-abcb-e2d8fb0c97f0)

Ce bouton modifie l’affichage du spectrogramme, vous permettant de zoomer sur une plage de fréquences spécifiée. Il n’altère pas l’audio envoyé au modèle pour analyse. Si vous souhaitez modifier l’audio analysé, vous devez appliquer des filtres audio dans le panneau des paramètres, puis sélectionner « Envoyer l’audio filtré pour l’analyse » dans la section Avancée des paramètres. Assurez-vous également de sélectionner le français comme langue, afin que l'aide soit traduite (ainsi que les noms des détections).

Le bouton avec les flèches :
![image](https://github.com/user-attachments/assets/3ba5c518-199b-40fc-975b-71ec33c979f3)

Active le « mode contexte », qui compare le résultat de l’analyse du clip audio central de 3 secondes avec deux clips de 3 secondes de part et d’autre, décalés de 1,5 seconde – un chevauchement de 50 %. Pour qu’une détection soit signalée, elle doit être détectée dans deux des trois fenêtres utilisées pour la comparaison. Cela permet de réduire le nombre de fausses détections positives.

**Comment by MartinWINDELS** [comment]
Thank you for your patience.
Now I will have him analyze the night of 11-12/01/2025. The results will be
visible on TREKTELLEN.
Best regards

Merci pour votre patience.
Là, je vais lui faire analyser la nuit du 11 au 12/01/2025.Les résultats
seront visibles sur TREKTELLEN.
Bien à vous.

Le sam. 11 janv. 2025 à 19:57, Mattk70 ***@***.***> a écrit :

> @MartinWINDELS <https://github.com/MartinWINDELS>, Merci pour votre
> explication.
>
> Lorsque vous analysez un long fichier audio, les modèles d'IA sont souvent
> désorientés, surtout dans les environnements urbains où il y a beaucoup de
> bruits de fond. De plus, les appels faibles ou inhabituels peuvent ne pas
> être détectés correctement. La fonctionnalité clé de Chirpity est qu'elle
> facilite grandement l'examen, le filtrage et la vérification des
> détections. Vous pouvez également comparer ce que l'IA pense être l'appel
> avec une sélection d'appels similaires provenant du site web Xeno-Canto.
>
> Je pense que vous parlez de ce bouton pour sélectionner une plage :
> image.png (view on web)
> <https://github.com/user-attachments/assets/f76889aa-4796-4104-abcb-e2d8fb0c97f0>
>
> Ce bouton modifie l’affichage du spectrogramme, vous permettant de zoomer
> sur une plage de fréquences spécifiée. Il n’altère pas l’audio envoyé au
> modèle pour analyse. Si vous souhaitez modifier l’audio analysé, vous devez
> appliquer des filtres audio dans le panneau des paramètres, puis
> sélectionner « Envoyer l’audio filtré pour l’analyse » dans la section
> Avancée des paramètres. Assurez-vous également de sélectionner le français
> comme langue, afin que l'aide soit traduite (ainsi que les noms des
> détections).
>
> Le bouton avec les flèches :
> image.png (view on web)
> <https://github.com/user-attachments/assets/3ba5c518-199b-40fc-975b-71ec33c979f3>
>
> Active le « mode contexte », qui compare le résultat de l’analyse du clip
> audio central de 3 secondes avec deux clips de 3 secondes de part et
> d’autre, décalés de 1,5 seconde – un chevauchement de 50 %. Pour qu’une
> détection soit signalée, elle doit être détectée dans deux des trois
> fenêtres utilisées pour la comparaison. Cela permet de réduire le nombre de
> fausses détections positives.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kahst/BirdNET-Analyzer/discussions/538#discussioncomment-11808008>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BNRRO3MRMYJCFR634VDTHC32KFSR3AVCNFSM6AAAAABUMLVECWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCOBQHAYDAOA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 


*Martin WINDELS*
*7780 COMINES, chaussée de Wervicq, 198*
***@***.**** ***@***.***>
*+32(0)474869508*

*MEMBRE CNB, NATAGORA, AVES, AVES MONS-TOURNAI, LRBPO, NATUURPUNT,
GON...MEMBRE du RESEAU NATURE : **1068P*
« Le silence des bêtes est la double expression de leur dignité et de notre
déshonneur. Nous autres, humains, faisons tant de vacarme. » Sylvain Tesson
« Je ne veux pas protéger la nature mais créer un monde où la nature
n'aurait pas besoin d'être protégée. »
« Mon pays est le monde et ma religion est de faire le bien. »  Thomas
Paine
J'y ajoute que mon peuple est l'Humanité.
« Le monde ne mourra pas par manque de merveilles mais uniquement par
manque d'émerveillement. »
« Ne doutez jamais qu'un petit groupe de gens réfléchis et engagés puisse
changer le monde. En fait, c'est toujours comme cela que ça s'est
passé. »  Margaret
Mead (1901 - 1978) - Anthropologue.
